packages/app/src/vis-packs/ValueLoader.tsx

```diff
@@ -42,13 +37,17 @@ function ValueLoader(props: Props) {
             <div />
             <div />
           </div>
-          {progress && (
-            <div className={styles.progressBars}>
-              {progress.slice(0, MAX_PROGRESS_BARS).map((val, index) => (
-                <progress className={styles.progress} key={index} value={val} /> // eslint-disable-line react/no-array-index-key
+          <div className={styles.progressBars}>
+            {[...ongoing.entries()]
+              .slice(0, MAX_PROGRESS_BARS)
+              .map(([key, val]) => (
+                <progress
+                  className={styles.progress}
+                  key={`${key.dataset.path}_${key.selection || ''}`}
```

Comment:
I now have access to the `Input` of the ongoing fetch — which, for the values store, is an object with a dataset and selection.